src/haddock/modules/topology/default.py

```diff
@@ -5,16 +5,45 @@
 from haddock.modules import BaseHaddockModule
 from haddock.structure import Molecule
 from haddock.pdbutil import PDBFactory
-from haddock.cns.topology import generate_topology
 from haddock.cns.engine import CNSJob, CNSEngine
+from haddock.cns.util import (load_workflow_params, generate_default_header,
+                              prepare_output, prepare_single_input)
 from haddock.ontology import ModuleIO, Format, PDBFile, TopologyFile
 from haddock.error import StepError
 from haddock.defaults import TOPOLOGY_PATH
 
-
 logger = logging.getLogger(__name__)
 
 
+def generate_topology(input_pdb, step_path, recipe_str, defaults,
+                      protonation=None):
+    """Generate a HADDOCK topology file from input_pdb"""
```

Comment:
I would suggest we start working on nice docstrings from scratch. I know this function was already implemented somewhere else but since we are moving it to another file lets use the opportunity to write proper docstrings. What is `input_put`, `step_path`, `recipe_str`, `defaults`, and `protonation`?

Reply:
We should probably decide on a format and implement it in the whole project, lets save it for another issue.

Reply:
I agree. I see these functions are very similar if not equal.